### DISCUSSION TITLE
account/billing page not loading

### DISCUSSION THREAD

**Discussion by hypnocill** [vercel/community#6830] [Account Issue]
### Question

I recently downgraded from Pro to Hobby and while everything seemed fine, I can see now the https://vercel.com/account/billing page doesn't load. I checked browser's dev tools and it seems that the '/plan' request fails with error {code: "missing_team_param"}.
This on its own is not bothering me now but I just want to be sure that my payment method is correctly setup so that my domains will renew successfully.
On the other hand, the billing page on my team: https://vercel.com/{team name}/~/settings/billing seems to be working fine.

**Comment by amyegan** [comment]
Thanks for reaching out about this. I'm not able to confirm payment details here, but if you [submit a case](https://vercel.com/help) with `Invoice Enquiry` selected as the problem area, that will get you in contact with the team that can check it for you. I'll let them know to look out for your case.
